x-pack/plugins/case/server/services/alerts/index.test.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { KibanaRequest } from 'kibana/server';
+import { elasticsearchServiceMock } from '../../../../../../src/core/server/mocks';
+import { CaseStatuses } from '../../../common/api';
+import { AlertService, AlertServiceContract } from '.';
+
+describe('updateAlertsStatus', () => {
+  const esClientMock = elasticsearchServiceMock.createClusterClient();
+
+  describe('happy path', () => {
+    let alertService: AlertServiceContract;
+
+    beforeEach(async () => {
+      alertService = new AlertService();
+      jest.restoreAllMocks();
+    });
+
+    test('it update the status of the alert correctly', async () => {
+      alertService.initialize(esClientMock);
+      await alertService.updateAlertsStatus({
+        ids: ['alert-id-1'],
```

Comment:
nit: this object repeats and can be made a variable. that way you can also reference the values in your `expect`